pkg/tenant/db/projects.go

```diff
@@ -66,16 +67,21 @@ func (p *Project) Validate() error {
 		return ErrMissingProjectName
 	}
 
+	if len(p.Description) > 2000 {
+		return ErrProjectDescriptionTooLong
```

Comment:
Can you make sure that this constraint is mentioned in the frontend story for project create?

Reply:
Edwin included the constraint when he created the FE story.